client/src/components/Booking/Booking.tsx

```diff
@@ -50,52 +20,44 @@ const Booking = ({ start, end, status, sitterId }: IRequest): JSX.Element => {
     setAnchorEl(null);
   };
 
-  const ye = new Intl.DateTimeFormat('en', { year: 'numeric' }).format(start);
-  const mo = new Intl.DateTimeFormat('en', { month: 'long' }).format(start);
-  const da = new Intl.DateTimeFormat('en', { day: '2-digit' }).format(start);
+  const year = new Intl.DateTimeFormat('en', { year: 'numeric' }).format(start);
+  const month = new Intl.DateTimeFormat('en', { month: 'long' }).format(start);
+  const day = new Intl.DateTimeFormat('en', { day: '2-digit' }).format(start);
   const startTime = new Intl.DateTimeFormat('en', { hour: 'numeric' }).format(start);
   const endTime = new Intl.DateTimeFormat('en', { hour: 'numeric' }).format(end);
-  const time = `${da} ${mo} ${ye}, ${startTime}-${endTime}`;
+  const time = `${day} ${month} ${year}, ${startTime}-${endTime}`;
 
   return (
     <Paper className={classes.root}>
       <Grid container spacing={1} direction="row">
         <Grid container xs={8} direction="column" className={classes.leftColumn}>
-          <Grid>
-            <Typography>{time}</Typography>
+          <Grid className={classes.timeGrid}>
+            <Typography className={classes.timeLabel}>{time}</Typography>
           </Grid>
-          <Grid container direction="row" spacing={1} className={classes.contactGrid}>
-            <Grid item>
-              <AccountCircleIcon fontSize="large" />
+          <Grid container direction="row" className={`${classes.contactGrid} ${classes.secondLinePadding}`}>
+            <Grid>
+              <Avatar />
             </Grid>
-            <Grid item>
-              <Typography>{sitterId}</Typography>
+            <Grid xs={1}></Grid>
+            <Grid>
+              <Typography className={classes.bookingSitterName}>{sitterId}</Typography>
             </Grid>
           </Grid>
         </Grid>
         <Grid container xs={4} direction="column" className={classes.rightColumn}>
           <Grid>
-            <Button
-              className={classes.settingsButton}
-              size="small"
-              aria-controls="simple-menu"
-              aria-haspopup="true"
-              onClick={handleClick}
-            >
+            <IconButton size="small" aria-controls="bookingSubmenu" aria-haspopup="true" onClick={handleClick}>
               <SettingsIcon fontSize="small" />
-            </Button>
-            <Menu id="simple-menu" anchorEl={anchorEl} keepMounted open={Boolean(anchorEl)} onClose={handleClose}>
+            </IconButton>
+            <Menu id="bookingSubmenu" anchorEl={anchorEl} keepMounted open={Boolean(anchorEl)} onClose={handleClose}>
```

Comment:
```suggestion
            <Menu id="booking-sub-menu" anchorEl={anchorEl} keepMounted open={Boolean(anchorEl)} onClose={handleClose}>
```
This id looks much better 🙌🏼 , though same casing conventions apply here as well.